setup.py

```diff
@@ -134,6 +134,9 @@ def make_pybind11_extension(
     rpathflag = '-Wl,-rpath,%s/lib' % sys.exec_prefix
     if "CONDA_PREFIX" in os.environ:
         include_dirs.append(os.path.join(os.environ["CONDA_PREFIX"], "include"))
+    else:
+        include_dirs.append(os.path.join("/usr/include/scotch"))
+        include_dirs.append(os.path.join("/usr/include/metis"))
```

Comment:
https://github.com/solvcon/solvcon/blob/master/libmarch/include/march/depend/scotch.hpp#L10 This line would cause build error if ``metis.h`` was not found. ``/usr/include/metis`` is trying to find ``metis.h``.

I may want to confirm the wording ``exclusive SCOTCH``. What does this mean actually? Does it mean SCOTCH which provides all related development files including ``metis.h``??